### ISSUE TITLE
Tidy up of decimal places in rain

### ISSUE THREAD

**Issue by 41south** [poblabs/weewx-belchertown#40]
This may already be on the list Pat, but there are varying values shown by the skin. Some are single decimal places and some are 2. My personal preference is 2 - as you will see in the attached screen shot my 0.46 rounds up to 0.5 etc

Also today is January 9th here but the records state 10 days without rain, think that should be 6in the Highest Year;
**Consecutive Days Without Rainfall | 6 days | ending January 07, 2019 | Highest Year | 10 days in 2019**

Something not quite right with the math there :)
![screen shot 2019-01-09 at 10 05 48 am](https://user-images.githubusercontent.com/1725001/50858948-d51f6d80-13f6-11e9-8caa-88ea9dca040b.png)

**Comment by poblabs** [comment]
Decimals is due to the hybrid of weewx code and my own code. I'll see if I can get it to match. 

The math on the days is a confusing one and it must have something to do with your timezone but I don't know why it would matter. Unless I need to specify the timestamp in your localtime (which I normally do but maybe forgot?). I'll see if I can take a look and try to replicate this.

**Comment by poblabs** [comment]
Feel free to grab the belchertown.py file from the commit above and add it to your system and restart weewx. The custom code I was doing for rain and outTemp had a bug where it wasn't honoring the decimal floats within the skin.conf. Oops 😃  Now it should match the rest of the weewx core floats based on my testing.

Still thinking about the other issue though.

**Comment by 41south** [comment]
Decimal places is fixed 👍

**Comment by poblabs** [comment]
The math on the days is actually valid. Turns out your days without rain started in December, and rolled across the new year. So the number of days are right, but the "highest year" is wrong. 

So I've made a change to get rid of "highest year" all together and just count the days, with the date being the ending day. 

This display now matches the current year records. 

![image](https://user-images.githubusercontent.com/3484775/52167590-709cb900-26eb-11e9-9d97-50fd5e2905a8.png)
